src/poliastro/twobody/orbit.py

```diff
@@ -271,7 +271,7 @@ def from_coords(cls, attractor, coord, plane=Planes.EARTH_EQUATOR):
         ----------
         attractor: Body
             Main attractor
-        coord: astropy.coordinates.SkyCoord or BaseCoordinateFrame
+        coord: astropy.coordinates.SkyCoord or ~astropy.coordinates.BaseCoordinateFrame
```

Comment:
Consistency issue here as well 
```suggestion
        coord: ~astropy.coordinates.SkyCoord or ~astropy.coordinates.BaseCoordinateFrame
```